src/screens/ForgotPassword/ForgotPassword.tsx

```diff
@@ -191,7 +190,7 @@ const ForgotPassword = (): JSX.Element => {
                     </div>
                     <Button
                       type="submit"
-                      className="mt-4 w-100"
+                      className={`mt-4 w-100 ${styles.login_btn}`}
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS.
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. There is a reference to `--bs-primary`

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296